### PR TITLE
Move AttachPeer entirely to the node

### PR DESCRIPTION
**Pull request by holisticode**
Closes https://github.com/ava-labs/avalanche-network-runner/issues/187 

Moved all AttachPeer stuff from the network to the local node.

### CODE REVIEW

File: local/node.go [hunk 1]
```diff
@@ -159,13 +161,26 @@ func (node *localNode) AttachPeer(ctx context.Context, router router.InboundHand
 			peerMsgQueueBufferSize,
 		),
 	)
+	cctx, cancel := context.WithTimeout(ctx, 30*time.Second)
```
Comment (felipemadero) [inline local/node.go:164]: constant for this?

**Review by felipemadero — COMMENTED** [review]
LGTM besides my comments and connor's one

File: server/server.go [hunk 2]
```diff
@@ -932,9 +944,9 @@ func (s *server) AttachPeer(ctx context.Context, req *rpcpb.AttachPeerRequest) (
 	s.mu.Lock()
 	defer s.mu.Unlock()
 
-	node, err := s.network.nw.GetNode((req.NodeName))
+	node, err := s.network.nw.GetNode(req.NodeName)
 	if err != nil {
-		return nil, err
+		return nil, ErrNodeNotFound
```
[the remark this answers is not in the document]
Comment (holisticode) [reply server/server.go:949]: I thought as we are getting a node, and it failed, we could just return ErrNodeNotFound. But yes, it's probably better to return the actual error. Fixed.

File: local/node.go (hunk 1 above)
Comment (holisticode) [reply local/node.go:164]: Added

**Review by felipemadero — APPROVED** [review]

**Pull request merged**